src/test/java/Interactors/LoadFileTest.java

```diff
@@ -46,7 +46,8 @@ void testCheckSaves() {
         // "cards.txt, properties.txt"
         String[] expectedReturn = new String[] {"cards.txt", "properties.txt"};
```

Comment:
This is a good addition if time permits, we could have it so that an error message shows up regarding corrupted game files so that the user will be prompted if these two files are not there.